caffe_app/consumers.py

```diff
@@ -41,6 +41,26 @@ def fetch_network_version(netObj):
     return network_version
 
 
+def update_data(data, required_data, version_id=None):
+    """
```

Comment:
Aren't these supposed to be single quotes?

Reply:
According to PEP 257 [here](https://www.python.org/dev/peps/pep-0257/), double quotes are to be used. Please check and confirm.

Reply:
We use single quotes, can you make the change.